snap/snapcraft.yaml

```
@@ -1,18 +1,18 @@
-name: nvidia-core22
-base: core22
+name: nvidia-2024
```

This is named after the Ubuntu version:
```suggestion
name: nvidia-2404
```